openedx/core/djangoapps/schedules/tasks.py

```diff
@@ -285,6 +286,7 @@ def _upgrade_reminder_schedules_for_bin(target_day, bin_num, org_list, exclude_o
 
             # This is used by the bulk email optout policy
             'course_ids': course_id_strs,
+            'cert_image': absolute_url(static('course_experience/images/verified-cert.png')),
```

Comment:
These emails are on a per-site basis. We might need to specific the base url to use for `absolute_url` based on the site that we're generating emails for (which means we might need to pass that down from the top of the task chain).

Reply:
I'm taking a stab now at addressing this.